packages/fx-core/src/common/tools.ts

```diff
@@ -372,15 +372,15 @@ export function isBicepEnvCheckerEnabled(): boolean {
 }
 
 export function isConfigUnifyEnabled(): boolean {
-  return isFeatureFlagEnabled(FeatureFlagName.ConfigUnify, false);
+  return isFeatureFlagEnabled(FeatureFlagName.ConfigUnify, true);
 }
 
 export function isInitAppEnabled(): boolean {
   return isFeatureFlagEnabled(FeatureFlagName.EnableInitApp, false);
 }
 
 export function isAadManifestEnabled(): boolean {
-  return isFeatureFlagEnabled(FeatureFlagName.AadManifest, false);
+  return isFeatureFlagEnabled(FeatureFlagName.AadManifest, true);
```

Comment:
Why change default value to true?

Reply:
OH, for test only, forget to remove it..